src/contract_wrappers/tokens/security_token_wrapper.ts

```diff
@@ -875,12 +944,22 @@ export default class SecurityTokenWrapper extends ERC20TokenWrapper {
   };
 
   public disableController = async (params: TxParams) => {
+    await this.checkOnlyOwner();
+    assert.assert(
+      await (await this.featureRegistryContract()).getFeatureStatus.callAsync(Features.DisableControllerAllowed),
```

Comment:
Disable controller is not a feature managed by Polymath in FeatureRegistry, It is a variable in the ST contract, so it should be checked there.